distribution/docs/src/main/resources/content/_integrating/_endpoints/ftp-endpoint.adoc

```diff
@@ -3,39 +3,28 @@
 :status: published
 :link: _ftp_endpoint
 :operations: ingest
-:url: \ftp://<FQDN>:8021/
 :summary: Ingests files directly into the ${branding} catalog using the FTP protocol.
-:implements: https://tools.ietf.org/html/rfc959[FTP] {external-link}
 
 == {title}
 
 The FTP Endpoint provides a method for ingesting files directly into the ${branding} catalog using the FTP protocol.
 
 The FTP endpoint can be accessed from any FTP client of choice.
 Some common clients are FileZilla, PuTTY, or the FTP client provided in the terminal.
-The default port number is *8021*. If FTPS is enabled with 2-way TLS, a client that supports client authentication is required.
+The default port number is *8021*.
+If FTPS is enabled with 2-way TLS, a client that supports client authentication is required.
 
-Custom Ftplets can be implemented by extending the `DefaultFtplet` class provided by Apache FTP Server.
-Doing this will allow custom handling of various FTP commands by overriding the methods of the `DefaultFtplet`.
-Refer to https://mina.apache.org/ftpserver-project/ftplet.html {external-link} for available methods that can be overridden.
-After creating a custom Ftplet, it needs to be added to the FTP server’s Ftplets before the server is started. Any Ftplets that are registered to the FTP server will execute the FTP command in the order that they were registered.
+=== FTP Endpoint Create examples
 
-.Operations
-[cols="2,1m,2,5m", options="header"]
-|===
+To ingest files into ${branding} send an FTP `PUT` request to the ${branding} server.
 
-|Operation
-|FTP Request Type
-|Details
-|Example URL
+.FTP Endpoint URL
+[source,ftp]
+----
+ftp://<FQDN>:<PORT>
+----
 
-|ingest
-|PUT
-|
-|{url}
+The FTP endpoint also supports the FTP `RNTO` operation to rename files as they are ingested.
 
-|===
-
-The FTP endpoint supports the `PUT`, `MPUT DELE`, `RETR`, `RMD`, `APPE`, `RNTO`, `STOU`, and `SITE` operations.
-
-The FTP endpoint supports files being uploaded as a dot-file (e.g., `.foo`) and then being renamed to the final filename (e.g., `some-file.pdf`). The endpoint will complete the ingest process when the rename command is sent.
+Files named with a leading `.`, such as `.<FILENAME>`, can be renamed in the ${branding} catalog to a new file name.
```

Comment:
Why do files with a leading . need to be renamed?
I assume it is because the ingester will ignore them.

Reply:
I'll check on this. My understanding was that, rather than ignore them, it would signify that they would be renamed

Reply:
I think it is something we should document.

Reply:
updated